packages/migration/migrations/application-config/20221205071415-add_health_facility_to_formdataset.ts

```diff
@@ -10,7 +10,9 @@
  * graphic logo are (registered/a) trademark(s) of Plan International.
  */
 
-export const up = async (db, client) => {
+import { Db, MongoClient } from 'mongodb'
+
+export const up = async (db: Db, client: MongoClient) => {
```

Comment:
We should perhaps use this type here: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/migrate-mongo/index.d.ts#L105